packages/frontend/component/src/components/block-suite-editor/index.tsx

```diff
@@ -129,6 +129,21 @@ const BlockSuiteEditorImpl = (props: EditorProps): ReactElement => {
     };
   }, [editor, page.awarenessStore, page.meta.trash, setBlockHub]);
 
+  useEffect(() => {
+    if (props.path?.length) {
+      const selectPath = props.path;
+      const selectManager = editor.root.value?.selection;
+      if (selectManager) {
+        setTimeout(() => {
```

Comment:
After https://github.com/toeverything/blocksuite/pull/5237 merged we will no need to use `setTimeout`.

```ts
useEffect(() => {
    if (props.path?.length) {
      const selectPath = props.path;
      editor.updateComplete
        .then(() => {
          const selectManager = editor.root.value?.selection;
          assertExists(selectManager);
          const path = selectManager.getInstance('block', { path: selectPath });
          selectManager.set([path]);
        })
        .catch(err => console.error(err));
    }
  }, [editor, props.path]);
```

Reply:
For unknown reasons this does not work, this will be fixed in the future.